### PR TITLE
Create ProcedureForIssues.md

### PR DESCRIPTION
**Pull request by Bombohog**

### CODE REVIEW

File: 09 Environment/ProcedureForIssues.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+# Procedure for creating issues
+
+
+*Here you will find a guide on how to create issues for projects (iterations), and how you can connect them to pull requests. There is a short text and a step by step guide which is longer.*
+
+
+### Kort
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:8]: This needs to be translated to correspond with the other titles

File: 09 Environment/ProcedureForIssues.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+
+- You have to remember to connect which project and milestone it is a part of on the right side.
+
+
+- After doing all that then click Submit new issue.
+
+<br/>
+
+###### Ekstra
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:62]: Also needs translation

File: 09 Environment/ProcedureForIssues.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+To create an issue fast then go into "Issues" and create a new issue, though you have to remember to choose what project it should be placed on and what milestone it is to be connected to, or you can go into the project and create a new note and then convert it into an issue.
+
+<br/>
+
+### Guide
+
+
+#### There exits two methods for creating Issues
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:18]: you probably mean "exists" and not "exits"

File: 09 Environment/ProcedureForIssues.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+
+# Procedure for creating issues
+
+
+*Here you will find a guide on how to create issues for projects (iterations), and how you can connect them to pull requests. There is a short text and a step by step guide which is longer.*
+
+
+### Kort
+
+
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:10]: Please add a paragraph about naming convention, if we do the same as HOEK its literally the name of the issue, just camel cased

**Review by MBRzealand — CHANGES_REQUESTED** [review]

File: 09 Environment/ProcedureForIssues.md [hunk 5]
```diff
@@ -59,7 +71,7 @@ To create an issue fast then go into "Issues" and create a new issue, though you
 
 <br/>
 
-###### Ekstra
+###### Exstra
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:74]: Sorry but it's "Extra"

File: 09 Environment/ProcedureForIssues.md [hunk 6]
```diff
@@ -2,10 +2,22 @@
 # Procedure for creating issues
 
 
+## Naming
+
+Start "Big" not "small" when naming the issue.
+
+If the issue is about creating an artifakt or feature then use "Create" as the first word.
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:9]: "Artifact"

File: 09 Environment/ProcedureForIssues.md [hunk 7]
```diff
@@ -2,10 +2,22 @@
...
+
+If the artefakt or feature exists already then use "Update" as the first word.
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:11]: artifact

File: 09 Environment/ProcedureForIssues.md [hunk 8]
```diff
@@ -2,10 +2,22 @@
...
+
+Give the issue a description if the title alone is not enough to describe what the issue is about and/or if the title is easy to misunderstand.
+
+## Creating
+
 *Here you will find a guide on how to create issues for projects (iterations), and how you can connect them to pull requests. There is a short text and a step by step guide which is longer.*
```
Comment (MBRzealand) [inline 09 Environment/ProcedureForIssues.md:17]: Are the " * * " intentional?

**Review by MBRzealand — CHANGES_REQUESTED** [review]

File: 09 Environment/ProcedureForIssues.md (hunk 8 above)
Comment (Bombohog) [reply 09 Environment/ProcedureForIssues.md:17]: yes

**Pull request merged**